messaging/src/test/scala/uk/ac/wellcome/messaging/worker/steps/MonitoringProcessorTest.scala

```diff
@@ -1,86 +0,0 @@
-package uk.ac.wellcome.messaging.worker.steps
```

Comment:
This has become messaging/src/test/scala/uk/ac/wellcome/messaging/worker/monitoring/metrics/MetricsMonitoringProcessorTest.scala I guess the change is too big for git to notice